core_impl/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -170,7 +170,7 @@ void record(TagContext tags, long value, Timestamp timestamp) {
     for (int i = 0; i < columns.size(); ++i) {
       TagKey tagKey = columns.get(i);
       if (!tags.containsKey(tagKey)) {
-        // replace not found key values by “unknown/not set”.
+        // replace not found tag values with null.
```

Comment:
Is this related to the other changes? If not I would prefer a separate pull. Regardless, this seems misleading--null suggests an empty string to me, not "UNKNOWN_TAG_VALUE".

Reply:
Sure, I'll make another PR to fix this.